### ISSUE TITLE
Reading `err` fails if nothing written to `stderr` in 0.0.9

### ISSUE THREAD

**Issue by jbasko** [kennethreitz/delegator.py#22]
0.0.9, Python 2.7 on Mac, `err` property is broken:

```
import delegator

# If you have something written to stderr, all is good
p1 = delegator.run('>&2 echo "error1"')
print p1.err

# This raises ValueError:
p2 = delegator.run('echo "error2"')
print p2.err
```

Output:

```
error1

Traceback (most recent call last):
  File "..../scratch_7.py", line 8, in <module>
    print p2.err
  File "..../.venv/lib/python2.7/site-packages/delegator.py", line 106, in err
    self.__err = self.std_err.read()
ValueError: I/O operation on closed file
```

This is ok in 0.0.8

**Comment by nateprewitt** [comment]
Hey @jbasko, this was fixed in ad441a2 and should be available in the next release. For the time being you'll need to use 0.0.8 or the master branch. Thanks for taking the time to check in on this.

**Comment by jbasko** [comment]
Cool, thanks @nateprewitt
